api/v1alpha1/imagepolicy_types.go

```diff
@@ -34,6 +34,11 @@ type ImagePolicySpec struct {
 	// selecting the most recent image
```

Comment:
Here's what I had in mind re: https://github.com/fluxcd/image-reflector-controller/issues/62#issuecomment-737546210